libs/search-widget/src/components/answer/DialogInput.svelte

```diff
@@ -38,6 +87,14 @@
     aria-label="Dialog input"
     bind:value={question}
     on:keypress={onKeyPress} />
+  {#if $isSpeechEnabled}
+    <div class="microphone">
+      <IconButton
+        icon="microphone"
+        aspect={$isSpeechStarted ? 'solid' : 'basic'}
```

Comment:
I guess what you want is to mark the button as active while the speech started. I feel like we should use the active state of the button instead of changing its aspect. 
`IconButton` (and `Button`) component already have an active style which is done by both the `:active` state of the button and an `.active` class. So you can easily add an `active` property on IconButton component to force the active state (it's something we already have on Angular buttons).